tests/system/general/test_27_Maxwell2D.py

```diff
@@ -105,6 +105,13 @@ def m2d_setup(add_app):
     app.close_project(app.project_name)
 
 
+@pytest.fixture()
+def m2d_analyze(add_app):
```

Comment:
@nunesanderson @anur7 please check because there should be already an m2d instance with an empty design. I don't think there is the need to create a new one.